src/token/IToken.sol

```diff
@@ -75,10 +75,16 @@ interface IToken is IUUPS, IERC721Votes, TokenTypesV1, TokenTypesV2 {
         address initialOwner
     ) external;
 
-    /// @notice Mints tokens to the auction house for bidding and handles founder vesting
+    /// @notice Mints tokens to the caller and handles founder vesting
     function mint() external returns (uint256 tokenId);
 
-    /// @notice Burns a token that did not see any bids
+    /// @notice Mints tokens to the recipient and handles founder vesting
+    function mint(address recipient) external returns (uint256 tokenId);
+
+    /// @notice Mints the specified amount of tokens to the recipient and handles founder vesting
+    function mint(uint16 amount, address recipient) external returns (uint256[] memory tokenIds);
```

Comment:
Thoughts on overloading vs choosing new fn names?

I see overloading less often with solidity but it should work just fine.

Reply:
I'm happy to rename the new ones. Want to keep the original function the same (i.e. `mint()`, the function that Auction calls) to avoid any compatibility issues for people migrating to 1.2.0.